### PR TITLE
temporum is not the only scanner that exists OR add scanners page

### PR DESCRIPTION
**Pull request by Semisol**
please edit the scanners/index.html to add your own scanner, thank you

### CODE REVIEW

**Review by SSouper — APPROVED** [review]
:3

**Review by RealY700 — CHANGES_REQUESTED** [review]
@NotNite @mat-1 @SSouper and @Semisol 
Add your scanner to this page, feel free to link anything related to your scanners and write whatever description you desire, or not include your scanner on the page

File: scanners/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        </p>
+        <div>
+          <h2>Temporum</h2>
+          <p>
+            <a href="https://temporum.org">Website</a><br><br>
+            This is a private Discord bot, it has an in game server browser and a streamsniper.
+          </p>
+        </div>
+        <div>
+          <h2>scanner</h2>
```
Comment (NotNite) [inline scanners/index.html:25]: man im gonna be real i just woke up and i am sick today can someone just do it for me and get chatgpt to write a description https://github.com/n2pm/loremipsum

**Review by NotNite — DISMISSED** [review]

File: scanners/index.html (hunk 1 above)
Comment (RealY700) [reply scanners/index.html:25]: yep, 1 sec

**Comment by adryd325** [comment]
Maybe a genric projects page that encompasses solutions to all the challenges in LO's CTF; and showcases our clients, and other tooling; not just scanning

**Comment by RealY700** [comment]
> Maybe a genric projects page that encompasses solutions to all the challenges in LO's CTF; and showcases our clients, and other tooling; not just scanning

For now a scanners page will do until we bother making one.
(I think we are all too lazy to convert this PR to be that, but it is a really good idea)

**Comment by adryd325** [comment]
> > Maybe a genric projects page that encompasses solutions to all the challenges in LO's CTF; and showcases our clients, and other tooling; not just scanning
> 
> For now a scanners page will do until we bother making one. (I think we are all too lazy to convert this PR to be that, but it is a really good idea)

I'll happily do that either tonight or when I get home from europe

**Review by RealY700 — APPROVED** [review]

**Comment by RealY700** [comment]
@NotNite please approve the new changes I guess
and @mat-1 if you ever want to add your scanner just make a commit later on, we'll be merging this soon

**Comment by NotNite** [comment]
I don't know where the approve button is on mobile, pretend I approved it

**Comment by RealY700** [comment]
:3

**Pull request merged**